.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,82 @@
 Hi! Before submitting your PR for review, and/or before merging it, please
-go through the following checklist:
+go through the following checklists:
 
-- [ ] For DB changes, check for missing indexes, check to see if the changes
-affect other apps (such as the dashboard), make sure the DB columns in the
-various environments are properly populated, coordinate with devops, plan
-migrations in separate steps.
+### General
 
-- [ ] For route changes, make sure GET requests don't change state or result in
-destructive behavior. GET requests should only result in information being
-read, not written.
+- [ ] `make lint` ran locally without errors
 
-- [ ] For encryption changes, make sure it is compatible with data that was
-encrypted with the old code.
+- [ ] Rubocop or Reek offenses were not disabled unless they are false positives.
+If you're not sure, please ask a teammate.
 
-- [ ] For secrets changes, [make sure to update the S3 secrets bucket](https://github.com/18F/identity-private/wiki/Secrets-S3-buckets) with the 
-new configs in **all** environments. 
+- [ ] Tests added for this feature/bug
 
-- [ ] Do not disable Rubocop or Reek offenses unless you are absolutely sure
-they are false positives. If you're not sure how to fix the offense, please
-ask a teammate.
+- [ ] The PR adheres to our [PR guidelines]
+
+- [ ] Related commits have been squashed
+
+- [ ] Method names in classes are in descending order of abstraction.
+See https://youtu.be/0rsilnpU1DU?t=554
+
+### Database
+
+- [ ] Unsafe migrations are implemented over several PRs and over several
```

Comment:
It might be useful to list the ordering of operations for safe migrations when moving data to new columns or tables. Something like:

1. create new column/table and write to it everywhere we write to the "old" column
2. after deploying (1), copy data from the "old" column(s) to the new column(s) or table(s)
3. after running (2), read data from the new column(s) or table(s) and optionally check that it matches the "old" data (e.g., using `scientist`)
4. after deploying (3) and verifying that the new column(s)/table(s) have the correct data, stop writing to the old column(s)
5. after deploying (4), clear out, remove, delete the old columns/tables as needed.

Reply:
This is essentially the same scenario as renaming a table, which is covered in the strong migrations documentation: https://github.com/ankane/strong_migrations#renaming-a-table